docs/getting-started/scalingsolutions.md

```diff
@@ -2,19 +2,19 @@
 title: Scaling UMA 
 sidebar_label: Scaling Solutions
 ---
-This section explains the how UMA is scaling to the Polygon network. Polygon is the first scaling solution enabled with more to follow soon.
+This section explains how UMA is scaling to other EVM networks. The full list of supported networks can be found [here](dev-ref/addresses.md).
 
 ## How scaling UMA works on Polygon
 
 The UMA ecosystem comprises a dispute resolution layer — the Data Verification Mechanism (DVM), and the financial contracts it secures. UMA’s architecture has from conception been decoupled, meaning the dispute layer and the contract layer do not need to live on the same chain.
 
-Having a decoupled architecture gives the advantage of being able to deploy to any EVM chain efficiently. Our scaling strategy allows the contract layer on Polygon or any EVM scaling solution to access the same DVM any contract native to Ethereum has access to.
+Having a decoupled architecture gives the advantage of being able to deploy to any EVM chain efficiently. Our scaling strategy allows the contract layer on any EVM chain to access the same DVM any contract native to Ethereum has access to.
 
-UMA makes use of an arbitrary message bridge that allows for two-way messages to pass between the Polygon network and Ethereum mainet. The message bridge uses the Polygon Arbitrary Message system, meaning it can be as trusted as the Polygon network itself. 
+UMA makes use of an arbitrary message bridge that allows for two-way messages to pass between the EVM network and Ethereum mainnet. The message bridge uses the scaling EVM network's Arbitrary Message system, meaning it can be as trusted as the EVM network itself. 
```

Comment:
```suggestion
UMA makes use of an arbitrary message bridge that allows for two-way messages to pass between the secondary EVM network and Ethereum mainnet. The message bridge typically uses the secondary network's Arbitrary Message system, meaning it can be as trusted as the EVM network itself. 
```